chapter06/rwd/ex2-6.pl

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Data::Dumper;
+
+my %words;
+
+print "Type words, one per line (CTRL-D to stop):\n";
+while (<STDIN>) {
+	chomp;
+	$words{$_}++;
+}
+
+print Data::Dumper->Dump([\%words], [qw(Sumary)]);
```

Comment:
wopsss I suppose that If I would sort it yes, but I didn't knew that it had to be sorted '^^